unit_test/CMakeLists.txt

```diff
@@ -159,7 +159,9 @@ Picasso_add_tests(MPI NAMES
   GridOperator3d
   GridOperator2d
   ParticleInterpolation
-  LevelSetRedistance
-  ParticleLevelSet
   MarchingCubes
   )
+
+if(Picasso_ENABLE_ARBORX)
+  list(APPEND MPI LevelSetRedistance ParticleLevelSet)
```

Comment:
This will never run because you call `Picasso_add_tests` before these tests are added to the list